Makefile

```diff
@@ -96,6 +96,14 @@ validate-step-registry:
 	$(CONTAINER_ENGINE) pull registry.ci.openshift.org/ci/ci-operator-configresolver:latest
 	$(CONTAINER_ENGINE) run --platform linux/amd64 --rm -v "$(CURDIR)/core-services/prow/02_config:/prow:z" -v "$(CURDIR)/ci-operator/config:/config:z" -v "$(CURDIR)/ci-operator/step-registry:/step-registry:z" registry.ci.openshift.org/ci/ci-operator-configresolver:latest --config /config --registry /step-registry --prow-config /prow/_config.yaml --validate-only
 
+refresh-bugzilla-prs:
+	mkdir -p /tmp/refresh-bugzilla-prs
```

Comment:
can you make this use the `TMPDIR` env var? and then default to `/tmp` if unset?